index.js

```diff
@@ -9,6 +9,7 @@ const cleanStack = require('clean-stack');
 const dotProp = require('dot-prop');
 const CacheConf = require('cache-conf');
 const updateNotification = require('./lib/update-notification');
+const {format} = require('./lib/utils');
```

Comment:
Destructuring is not yet supported in Node 4. So just import it as `utils` and use `utils.format` below.